### PR TITLE
authenticate to GitHub API

### PR DESCRIPTION
**Pull request by JoeArmani**
Related: https://github.com/episphere/connect/issues/900

Authenticate requests from GitHub API (raises limit from 60/hour to 5000/hour).
Use: SHA codes for survey modules.

### CODE REVIEW

File: utils/connectApp.js [hunk 1]
```diff
@@ -57,6 +57,23 @@ const connectApp = async (req, res) => {
 
     else if (api === 'validateEmailOrPhone') return validateUsersEmailPhone(req, res);
 
+    else if (api === 'getModuleSHA') {
+      if(req.method !== 'GET') {
```
Comment (Gbarra9) [inline utils/connectApp.js:61]:
```suggestion
      if (req.method !== 'GET') {
```

**Review by Gbarra9 — APPROVED** [review]
Looks good. I was unaware about the github limits.

File: utils/connectApp.js (hunk 1 above)
Comment (JoeArmani) [reply utils/connectApp.js:61]: Thanks, updated.

**Review by bransteitterbr — APPROVED** [review]

**Review by anthonypetersen — APPROVED** [review]
looks great

**Pull request merged**